src/components/Modal/PModal.vue

```diff
@@ -60,29 +59,34 @@
   import { nextTick, computed, ref, useSlots, watch, onBeforeUnmount } from 'vue'
   import PButton from '@/components/Button/PButton.vue'
   import PIcon from '@/components/Icon/PIcon.vue'
+  import { useGlobalEventListener, useModal } from '@/compositions'
   import { useFocusableElements } from '@/compositions/useFocusableElements'
   import { keys } from '@/types'
   import { Icon } from '@/types/icon'
+  import { isKeyEvent } from '@/utilities'
 
   const props = defineProps<{
-    showModal: boolean,
+    showModal?: boolean,
     title?: string,
     icon?: Icon,
     autoClose?: boolean,
   }>()
 
-  const emits = defineEmits<{
+  const emit = defineEmits<{
     (event: 'update:showModal', value: boolean): void,
   }>()
 
-  const internalValue = computed({
+  const internalValue = ref<boolean>(false)
+  const modelValue = computed<boolean>({
```

Comment:
same tweak here. I know these were already named this but might as well updated them to latest naming convention.